lib/services/networkManagement2/package.json

```diff
@@ -6,7 +6,7 @@
     "Rajendran, Deepak <derajen@microsoft.com>",
     "Choubey, Ankur <anchoube@microsoft.com>"
   ],
-  "version": "1.3.0-preview",
+  "version": "1.4.0-preview",
```

Comment:
@DeepakRajendranMsft - I think you should weigh in on what changes this API brings, rather than Anton and me guessing if this is a breaking change or not 😃